appcontainer/ViewportSizeModel.js

```diff
@@ -42,27 +34,41 @@ export class ViewportSizeModel extends HoistModel {
         makeObservable(this);
         window.addEventListener('resize', () => this.setViewportSize());
         this.setViewportSize();
+
+        this.addReaction({
+            track: () => this.size,
+            run: () => this.updateOrientation(),
+            fireImmediately: true,
+            // Further debounce (on top of debounced setViewportSize) to allow for even greater
+            // lag observed on mobile device responses to DOM orientation APIs.
+            debounce: 100
```

Comment:
Curious about this second level of debouncing here - is this just to allow for the window.screen / orientation APIs to update? Wondering if we should drop them and only calc isPortrait from aspect ratio, if using them is actively degrading the experience like this?